Entity/BasePostRepository.php

```diff
@@ -12,6 +12,7 @@
 namespace Sonata\NewsBundle\Entity;
 
 use Doctrine\ORM\EntityRepository;
+use Doctrine\ORM\QueryBuilder;
```

Comment:
I think this is a bit awkard, but we will need these statements when using php7 type hinting, so ok.